testdata/configroot/scopes/global/descriptors.yml

```diff
@@ -16,6 +16,8 @@ manifests:
         valueType: STRING
       source.serviceAccount:
         valueType: STRING
+      source.user:
```

Comment:
i just noticed that this in the kubernetes manifest... 

let's move it to the istio-proxy section (here and in the other file).

Reply:
Just followed other source.*. Done.